pkg/agent/bakerapi.go

```diff
@@ -101,7 +101,7 @@ func (agentBaker *agentBakerImpl) GetNodeBootstrappingForScriptless(
 	}
 
 	nodeBootstrapping := &datamodel.NodeBootstrapping{
-		CSE:        "",
+		CSE:        scriptlessBootstrapStatusCSE,
```

Comment:
nit: keep naming consistent for customData as well, scriptlessCustomData.

if not, then remove the same from CSE, your call